app/views/admin/reports/index.html.haml

```diff
@@ -14,7 +14,9 @@
             = I18n.t(:name, scope: [:admin, :reports, report_type])
           %td
             - if report_subtypes.empty?
-              - name = I18n.t(:name, scope: [:admin, :reports, report_type])
-              - url = main_app.admin_report_url(report_type: report_type)
-              = link_to name, url
+              %ul{style: "margin-left: 12pt"}
+                %li
+                  - name = I18n.t(:name, scope: [:admin, :reports, report_type])
+                  - url = main_app.admin_report_url(report_type: report_type)
+                  = link_to name, url
```

Comment:
I like that it's now the same layout. But shouldn't we re-use the same partial then?
```suggestion
              - name = I18n.t(:name, scope: [:admin, :reports, report_type])
              - url = main_app.admin_report_url(report_type: report_type)
              = render partial: "report_subtype", locals: { report_subtypes: [[name, url]], report_type: report_type }
```

Or even better, we just set the variable and it's rendered below:

```suggestion
              - name = I18n.t(:name, scope: [:admin, :reports, report_type])
              - url = main_app.admin_report_url(report_type: report_type)
              - report_subtypes = [[name, url]]
```

I would prefer this to be set in the controller but I can't figure that out quickly. It's not that obvious.

Reply:
I do prefer the solution of @mkllnk and I've commited the solution: e16c8661a

Reply:
This was just a theoretical solution and tests failed. I now added another commit to fix this. :smile: